docs/serverless/assistant/connect-to-vertex.mdx

```diff
@@ -0,0 +1,68 @@
+---
+slug: /serverless/security/connect-to-google-vertex
+title: Connect to Google Vertex
+description: Set up a Google Vertex LLM connector.
+tags: ["security", "overview", "get-started"]
+status: in review
+---
+
+# Connect to Google Vertex AI
+
+This page provides step-by-step instructions for setting up a Google Vertex AI connector for the first time. This connector type enables you to leverage Vertex AI's large language models (LLMs) within ((kib)). You'll first need to enable Vertex AI, then generate an API key, and finally configure the connector in ((kib)).
+
+<DocCallOut title="Important"> 
+Before continuing, you should have an active project in one of Google Vertex AI's [supported regions](https://cloud.google.com/vertex-ai/docs/general/locations#feature-availability). 
+</DocCallOut>
+
+## Emable the Vertex AI API
+
+1. Log in to the GCP console and navigate to **Vertex AI → Vertex AI Studio → Overview**. 
+2. If you're new to Vertex AI, the **Get started with Vertex AI Studio** popup appears. Click **Vertex AI API**, then click **ENABLE**.
+
+The following video demonstrates these steps.
+
+<DocVideo source="vidyard" id="vFhtbiCZiKhvdZGy2FjyeT"/>
+
+For more information about enabling the Vertex AI API, refer to [Google's documentation](https://cloud.google.com/vertex-ai/docs/start/cloud-environment).
+
+
+## Create a Vertex AI service account
+
+1. In the GCP console, navigate to **APIs & Services → Library**.
+2. Search for **Vertex AI API**, select it, and click **MANAGE**.
+3. In the left menu, navigate to **Credentials** then click **+ CREATE CREDENTIALS** and select **Service account**.
+4. Name the new service account then click **CREATE AND CONTINUE**. 
+5. Under **Select a role**, select **Vertex AI User**, then click **CONTINUE**.
+6. Click **Done**.
+
+The following video demonstrates these steps.
+
+<DocVideo source="vidyard" id="tmresYYiags2w2nTv3Gac8"/>
+
+## Generate an API key
+
+1. Return to Vertex AI's **Credentials** menu and click **Manage service accounts**.
+2. Search for the service account you just created, select it, then click the link that appears under **Email**.
+3. Go to the **KEYS** tab, click **ADD KEY**, then select **Create new key**.
+4. Select **JSON**, then click **CREATE** to download the key. Keep it somewhere secure.
+
+The following video demonstrates these steps.
+
+<DocVideo source="vidyard" id="hrcy3F9AodwhJcV1i2yqbG"/>
+
+## Configure the Google Gemini connector
+
+Finally, configure the connector in ((kib)):
+
+1. Log in to ((kib)).
+2. Navigate to **Stack Management → Connectors → Create Connector → Google Gemini**.
+3. Name your connector to help keep track of the model version you are using.
+4. The **URL** field can be left as default.
+5. Enter your **GCP Region** and **GCP Project ID**.
+6. Under **Default model**, specify which [model](https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models) you want to use.
```

Comment:
Change to: Under Default Model, specify either:
gemini-1.5-pro
Gemini-1.5-flash

Note: for further information on model differences go [here](https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models).